src/System.Private.CoreLib/shared/System/Runtime/InteropServices/Marshal.Windows.cs

```diff
@@ -84,24 +84,48 @@ internal static unsafe int StringToAnsiString(string s, byte* buffer, int buffer
         // Returns number of bytes required to convert given string to Ansi string. The return value includes null terminator.
         internal static unsafe int GetAnsiStringByteCount(ReadOnlySpan<char> chars)
         {
-            fixed (char* pChars = chars)
+            int byteLength;
+
+            if (chars.Length == 0)
+            {
+                byteLength = 0;
```

Comment:
Should this be `1` ("the return value includes null terminator") or is that a special case?

Reply:
Oh, I see now.